Src/coffeescript/cql-execution/src/elm/datetime.coffee

```diff
@@ -135,6 +135,11 @@ module.exports.doAfter = (a, b, precision) ->
 module.exports.doBefore = (a, b, precision) ->
   a.before b, precision
 
+module.exports.doAddition = (a,b) ->
+
+module.exports.doSubtraction = (a,b) ->
+
```

Comment:
They do not do anything .  That was where I was originally going to put some of the functionality for date - quantify math.